crates/rpc/src/handlers/syncing.rs

```diff
@@ -0,0 +1,83 @@
+use actix_web::{HttpResponse, Responder, get, web::Data};
+use ream_beacon_api_types::{
+    error::ApiError,
+    responses::{DataResponse, EXECUTION_OPTIMISTIC},
+    sync::SyncStatus,
+};
+use ream_execution_engine::ExecutionEngine;
+use ream_fork_choice::store::Store;
+use ream_storage::{db::ReamDB, tables::Table};
+use serde::{Deserialize, Serialize};
+use tracing::error;
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct Syncing {
+    sync_status: SyncStatus,
+}
+
+impl Syncing {
+    pub fn new(head_slot: u64, sync_distance: u64, el_offline: bool, is_syncing: bool) -> Self {
+        Self {
+            sync_status: SyncStatus {
+                head_slot,
+                sync_distance,
+                is_syncing,
+                el_offline,
+                // TODO
```

Comment:
can a description be added here or just remove this todo, as we will know to fix this when we remove the constant